tests/models/sam_hq/test_processor_samhq.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+
+from transformers.testing_utils import require_torch, require_torchvision, require_vision
+from transformers.utils import is_torch_available, is_vision_available
+
+from ...test_processing_common import prepare_image_inputs
+
+
+if is_vision_available():
+    from PIL import Image
+
+    from transformers import AutoProcessor, SamHQProcessor, SamImageProcessor
+
+if is_torch_available():
+    import torch
+
+
+@require_vision
+@require_torchvision
+class SamHQProcessorTest(unittest.TestCase):
+    def setUp(self):
```

Comment:
Needs to have the processor_class defined here, as in `test_processor_sam.py` and other similar processor tests.
```suggestion
    processor_class = SamHQProcessor
    def setUp(self):
```